tests/unit/utils/test_auth_dependency.py

```diff
@@ -0,0 +1,80 @@
+"""Unit tests for auth_dependency module."""
+
+import os
+from unittest.mock import patch
+
+import pytest
+from fastapi import HTTPException, Request
+from fastapi.testclient import TestClient
+
+from ols.utils import config
+from ols.utils.auth_dependency import auth_dependency
+from tests.mock_classes.mock_k8s_api import (
+    mock_subject_access_review_response,
+    mock_token_review_response,
+)
+
+
+@patch.dict(os.environ, {"OLS_CONFIG_FILE": "tests/config/auth_config.yaml"})
+def setup():
+    """Setups and load config."""
+    global client
+    cfg_file = os.environ.get("OLS_CONFIG_FILE", "tests/config/auth_config.yaml")
+    config.init_config(cfg_file)
```

Comment:
why do you need these 2 lines?  the other tests don't need them to setup the config yaml path

Reply:
Resolved removed additional unneeded line